itests/hive-unit/src/test/java/org/apache/hadoop/hive/ql/parse/TestReplicationScenariosAcidTablesBootstrap.java

```diff
@@ -193,7 +193,7 @@ public void testAcidTablesBootstrapDuringIncrementalWithOpenTxnsTimeout() throws
     // t1=5+2(insert) and t2=5+5(insert, alter add column)
```

Comment:
can you update this comment on why t2=5+6 now?

Reply:
Done